CHANGES/644.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed type annotations for keys
```

Comment:
What keys? This is what will show up in the changelog, it seems like it doesn't have enough context/precision to be understood by the people reading it.
Also, add a trailing period so that it's a complete sentence.

Reply:
"for all multidict mapping classes"?